src/hooks/FreelyTransferable.sol

```diff
@@ -27,10 +27,10 @@ contract FreelyTransferable is BaseTransferHook {
         returns (bool)
     {
         if (isSourceOrTargetFrozen(from, to, hookData)) return false;
-        if (isDepositRequest(from, to)) return isTargetMember(to, hookData);
+        if (isIssuance(from, to)) return isTargetMember(to, hookData);
```

Comment:
Question: We no longer have a hook when the user deposits the request?

Reply:
We never did for `requestDeposit` because that just transfers assets from the user to the global escrow. The hooks act as a share token filter.

Reply:
Yeah I think we need to do that. But `isRedeemRequestOrRevocation` => `isRedeemClaimOrRevocation`